src/platform/forms/components/review/SavableErrorMessage.jsx

```diff
@@ -0,0 +1,119 @@
+// libs
```

Comment:
We still need to break up this component to be more modular. This is more of the first steps to get there.